contrib/utilities/update-copyright.sh

```diff
@@ -31,7 +31,7 @@ files="
          contrib/python-bindings/*/*.{h,cc,py} \
          contrib/utilities/{update-copyright,indent} \
          doc/doxygen/*/*.{h,h.in} \
-         doc/doxygen/scripts/{create_anchors,filter,intro2toc,program*,*.pl} \
+         doc/doxygen/scripts/{*.pl} \
```

Comment:
Not wrong, but you can remove the curly braces. (They used to enclose a comma-separated enumeration, but now there is just one case left.)

Reply:
Right, it just lined up so nicely that I didn't think about it.
It is updated now.